src/elements/Button/ButtonTheme.ts

```diff
@@ -59,9 +59,9 @@ export interface ButtonTheme {
 }
 
 const baseTheme: Partial<ButtonTheme> = {
-  base: 'inline-flex whitespace-no-wrap select-none items-center justify-center px-2.5 py-1 rounded',
+  base: 'inline-flex whitespace-no-wrap select-none items-center justify-center px-2.5 py-1 rounded-sm font-sans',
```

Comment:
@amcdnl Do we need define font on component level?
I think it should be defined on body element once

Reply:
I was having issues with Buttons and the default font. Normally a reset handles these so we probably need this.